lighthouse-core/audits/unused-css-rules.js

```diff
@@ -127,7 +132,7 @@ class UnusedCSSRules extends Audit {
     const unusedRatio = (unused / usage.length) || 0;
     const results = Object.keys(indexedSheets).map(sheetId => {
       return UnusedCSSRules.mapSheetToResult(indexedSheets[sheetId]);
-    });
+    }).filter(Boolean);
```

Comment:
Don't feel strongly about it, but `.filter(n => n)` would also filter out the falsey values.